Buildings/Controls/OBC/ASHRAE/G36/Atomic/Validation/SystemRequestsReheatBox.mo

```diff
@@ -0,0 +1,115 @@
+within Buildings.Controls.OBC.ASHRAE.G36.Atomic.Validation;
+model SystemRequestsReheatBox
+  "Validate model of generating system requests"
```

Comment:
This test does not validate the output for the boiler plant request. We need to include this output in the test results.